### PR TITLE
Add an editor config for easier colaboration

### PR DESCRIPTION
**Pull request by bassettsj**
- Add a simple editor editorconfig

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Set default charset
+charset = utf-8
+
+# 2 space indentation
+tab_size = 2
+tab_style = space
+
+# trim any trailing whitespace
+trim_trailing_whitespace = true
```
Comment (jeffmo) [inline .editorconfig]: Is there anyway to whitelist specific files for this? For example, I think this has bitten me on `.md` files in the past?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Unix-style newlines with a newline ending every file
+end_of_line = lf
+insert_final_newline = true
+
+# Set default charset
+charset = utf-8
+
+# 2 space indentation
+tab_size = 2
+tab_style = space
```
Comment (jeffmo) [inline .editorconfig]: tabs_v_spaces

Comment (bassettsj) [inline .editorconfig]: 😆  LOL that is correct for this project right?

File: .editorconfig (hunk 1 above)
Comment (bassettsj) [inline .editorconfig]: Yeah, we could just add that to js / json files let me change that.

Comment (bassettsj) [inline .editorconfig]: Changed :-)

File: .editorconfig (hunk 2 above)
Comment (jeffmo) [inline .editorconfig]: Hehe, yea :p

**Pull request merged**